vpr/src/route/route_timing.cpp

```diff
@@ -355,7 +356,11 @@ bool try_timing_driven_route(const t_router_opts& router_opts,
 
     route_budgets budgeting_inf;
 
-    auto router_lookahead = make_router_lookahead(router_opts.lookahead_type);
+    auto router_lookahead = make_router_lookahead(
```

Comment:
In line with one of the comments from #707, it looks like this will re-build the lookahead each time the router is invoked, which is not what we want. I think the lookahead (since it is now computed/loaded in make_router_lookahead) should be built externally and passed into the router.